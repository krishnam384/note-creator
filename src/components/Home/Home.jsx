import React from "react";

export default function Home() {
  const name = localStorage.getItem("username");
  return (
    <div>
      <h1>Welcome {name}</h1>
    </div>
  );
}
