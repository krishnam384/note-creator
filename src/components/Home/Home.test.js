import { shallow } from "enzyme";
import React from "react";
import Home from "./Home";

/* eslint no-undef: 0 */

const home = shallow(<Home />);
it("First snapshot", () => {
  expect(home).toMatchSnapshot();
});
