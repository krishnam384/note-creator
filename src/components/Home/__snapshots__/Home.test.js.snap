// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot 1`] = `
<div>
  <h1>
    Welcome 
  </h1>
</div>
`;
